cognite/powerops/resync/models/v2/dms/data_models/views/functions/dayAheadBacken-TaskDispatcherShopOutput.view.yaml

```diff
@@ -0,0 +1,83 @@
+space: '{{day_ahead_backend_space}}'
+externalId: TaskDispatcherShopOutput
+description: Input to the task dispatcher in the shop bid process
+name: TaskDispatcherShopOutput
+version: '{{day_ahead_backend_task_dispatcher_shop_output_version}}'
+implements:
+  - space: '{{day_ahead_backend_space}}'
+    externalId: FunctionInput
+    version: '{{day_ahead_backend_function_output_version}}'
+    type: view
+filter:
+  equals:
+    property:
+    - node
+    - property
+    - functionName
+    value: TaskDispatcherShop
+properties:
+  functionName:
+    container:
+      space: '{{day_ahead_backend_space}}'
+      externalId: FunctionMetadata
+      type: container
+    containerPropertyIdentifier: functionName
+    name: functionName
+    description: The function name
+  processId:
+    container:
+      space: '{{day_ahead_backend_space}}'
+      externalId: FunctionMetadata
+      type: container
+    containerPropertyIdentifier: processId
+    name: processId
+    description: The id of the dayahead bid shop process
+  processStep:
+    container:
+      space: '{{day_ahead_backend_space}}'
+      externalId: FunctionMetadata
+      type: container
+    containerPropertyIdentifier: processStep
+    name: processStep
+    description: The step in the bid process as an integer ex. 0,1,2
+  linkedStep:
+    container:
+      space: '{{day_ahead_backend_space}}'
+      externalId: FunctionMetadata
+      type: container
+    containerPropertyIdentifier: linkedStep
+    name: previousStep # here is where you map the name - the name that is shown in the view
+    description: The id of the dayahead bid shop process
+  alerts:
+    type:
+      space: '{{type_space}}'
+      externalId: calculationIssue
+    source:
+      space: '{{shared_model_space}}'
+      externalId: Alert
+      version: '{{base_Alert_version}}'
+      type: view
+    direction: outwards
+    name: alerts
+    description: An array of calculation level Alerts.
+    connectionType: multiEdgeConnection
+  preprocessorCalculations:
+    container:
+      space: '{{day_ahead_backend_space}}'
+      externalId: FunctionMetadata
+      type: container
+    containerPropertyIdentifier: data
+    name: preprocessorCalculations
+    description: The preprocessed and ready scenario to send to cogshop
+    direction: outwards
+    connectionType: multiEdgeConnection
+  partialBidCalculations:
```

Comment:
Is this the correct way? or could I point to a view of a `PartialBidCalculationInput` instead somehow?
<img width="468" alt="Screenshot 2024-01-25 at 14 53 06" src="https://github.com/cognitedata/power-ops-sdk/assets/47179573/bfe83fdd-4d84-472a-8313-76f85cac7650">

Reply:
Instead of some generic data from the FunctionMeta container